src/main/java/core/basesyntax/service/filewriter/FileWriterImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.filewriter;
+
+import core.basesyntax.dao.TransactionDao;
+import core.basesyntax.dao.TransactionDaoImpl;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class FileWriterImpl implements FileWriter {
+    private static final TransactionDao transactionDao = new TransactionDaoImpl();
+
+    @Override
+    public void writeToFile() {
+        File file = new File("src/main/resources/report.csv");
```

Comment:
```suggestion
    public void writeToFile(String filename) {
```

Reply:
filename should be constant, but this method would be more flexible if it had filename as an argument